features/shopcarts.feature

```diff
@@ -7,3 +7,15 @@ Scenario: The server is running
     When I visit the "home page"
     Then I should see "Shopcart Demo REST API Service"
     Then I should not see "404 Not Found"
+
+Scenario: List all shopcarts
+    When I visit "shopcarts"
+    Then I should see "Settlers of Catan"
```

Comment:
we are list ALL shopcart, aren't we? If you really want to make sure we have the correct list, we should get the JSON object and very information based on that. Verifying a uid is not different than verifying a product name or a product sku.